csrc/fusion.cpp

```diff
@@ -797,28 +797,37 @@ void Fusion::aliasOutputToInput(
     }
   }
 
-  NVF_ERROR(
-      isAliasCompatible(input, output),
-      "The input and output values are not alias-compatible.");
-  // Let integration hide any output that wasn't a fusion output when
-  // `aliasOutputToInput` was called. For example, running mean and var for
-  // batch norm.
-  io_alias_[output] = AliasInfo{
-      .type = type,
-      .aliased_io = input,
-      .hide_output = !output->isFusionOutput()};
+  // AllocationType::Evaluate
```

Comment:
This is not accurate. `::ReuseBuffer` reaches here too, and that's intended.

Reply:
Right. Will move it.